components/Session/Voting/Review/Content.js

```diff
@@ -0,0 +1,210 @@
+        }
+      }
+    }
+  }
+`;
+
+const CAST_VOTE = gql`
+  mutation castVote($eventId: ID!, $vote: VoteInput!) {
+    sessions {
+      voting(eventId: $eventId) {
+        cast(vote: $vote) {
+          id
+          notes
+          memberId
+        }
+      }
+    }
+  }
+`;
+
+const Content = () => {
+  const {
+    loading: sessionsLoading,
+    error: sessionsError,
+    data: sessionsData,
+  } = useQuery(GET_SESSIONS, {
+    variables: { eventId: process.env.CURRENT_EVENT_ID },
+  });
+
+  if (sessionsLoading) return null;
+  if (sessionsError) return null;
+
+  const root = sessionsData.sessions.me.voting;
+
+  let toastId = null;
+
+  const { totalSubmitted } = root;
+  const rootSessions = root.voted;
+  const totalRemaining = totalSubmitted - rootSessions.length;
+  const totalVotedOn = rootSessions.length;
+
+  const [sessions, setStateSessions] = useState(rootSessions);
+
+  const forceUpdate = () => {
+    ButterToast.dismiss(toastId);
+    ButterToast.raise({
+      sticky: false,
+      content: (
+        <Cinnamon.Crisp
+          scheme={Cinnamon.Crisp.SCHEME_BLUE}
+          content={() => <div>Vote successfully cast!</div>}
+          title="Vote"
+        />
+      ),
+    });
+    setStateSessions(sessions);
+  };
+
+  const [castVote] = useMutation(CAST_VOTE, {
+    onCompleted: forceUpdate,
+    onError: createError => {
+      throw new Error(createError);
+    },
+  });
+
+  const submitVote = (yesVote, id) => {
+    const index = sessions.findIndex(s => s.sessionId === id);
+    if (sessions[index].vote !== yesVote) {
+      sessions[index].vote = yesVote;
+      toastId = ButterToast.raise({
+        sticky: false,
+        content: (
+          <Cinnamon.Crisp
+            scheme={Cinnamon.Crisp.SCHEME_BLUE}
+            content={() => <div>Casting Vote...</div>}
+            title="Vote"
+          />
+        ),
+      });
+      const vars = {
+        eventId: process.env.CURRENT_EVENT_ID,
+        vote: {
+          sessionId: id,
+          vote: yesVote,
+        },
+      };
+      castVote({
+        variables: vars,
+      });
+    }
+  };
+
+  const converter = new MarkdownIt();
+  return (
+    <MainContent>
+      <NavLinks
+        forwardLabel="Continue Voting"
+        forwardLink="/wi/session/voting/vote"
+      />
+      <SessionsContainer>
+        {_.sortBy(sessions, s => s.title.toLowerCase()).map(s => {
+          return (
+            <SessionContainer key={s.id}>
+              <h4>{s.title}</h4>
+              <div>{parse(converter.render(s.longDescription))}</div>
+              <Section>
+                <DetailsHeader>Key Takeaways</DetailsHeader>
+                <ul>
+                  {s.takeaways.map(t => {
+                    return (
+                      <React.Fragment key={t}>
+                        <li>{t}</li>
+                      </React.Fragment>
+                    );
+                  })}
+                </ul>
+              </Section>
+              <Thumbs
```

Comment:
I opened a Clubhouse story for @brandonmartinez to check out the thumb svg and give us another if has a chance to get us a better thumb.